glide.lock

```diff
@@ -0,0 +1,64 @@
+hash: 6b4ef86ba3b26a4ff92055f8700395aef07af84c7f97df8642cb8f6459097036
```

Comment:
@pchico83 `glide.lock` is meant to be commited yes — it makes the build reproductible and "freeze" dependency version (I don't like `glide` very much but that's another *personal* store 😂 👼 ).